pkg/cmd/pulumi/new.go

```diff
@@ -254,6 +250,14 @@ func runNew(args newArgs) error {
 	proj.Name = tokens.PackageName(args.name)
 	proj.Description = &args.description
 	proj.Template = nil
+	// Hack for python, most of our templates don't specify a venv but we want to use one
```

Comment:
s/Hack/Workaround/

Reply:
I think this actually will break some users, like Nix users. There are situations where users explicitly don't want to use a venv. If Python now always uses venv even when not called for, we should highlight this as a change in CHANGELOG.

Reply:
Ah cool I was going to ask if we could just assume "venv" but that's a good reason not to. At any rate this isn't a breaking change because we already always set venv in "pulumi new"

Reply:
IN that case it's completely fine ofc.